aslam_cv_calibration/include/aslam/calibration/target-algorithms.h

```diff
@@ -11,13 +11,13 @@ namespace calibration {
 
 // Estimates the target transform from the given corner observations.
 bool estimateTargetTransformation(
-    const TargetObservation& target_observation,
+    const TargetObservation::ConstPtr& target_observation,
```

Comment:
i would prefer to leave it as a const& as it is more general (you can also call it if you own the object locally)